css-view-transitions-2/Overview.bs

```diff
@@ -656,20 +684,38 @@ When capturing the old or new state for an element, perform the following steps
 	<div algorithm>
 		To <dfn export>resolve cross-document view-transition</dfn> for {{Document}} |document|:
 
-		1. Let |transition| be |document|'s [=active view transition=].
+		1. [=Assert=]: |document| is [=fully active=].
+
+		1. Let |inboundViewTransitionParams| be |document|'s [=inbound view transition params=].
+
+		1. If |inboundViewTransitionParams| is null, then return null.
+
+		1. Set |document|'s [=inbound view transition params=] to null.
 
-		1. If |transition| is null or |transition|'s [=ViewTransition/is inbound cross-document transition=] is false,
-			then return null.
+		1. If |document|'s [=active view transition=] is not null, then return null.
 
-			Note: |transition|'s [=ViewTransition/is inbound cross-document transition=] would be false if a same-document
-			transition was started before the {{Window/pagereveal}} event was fired.
+			Note: this means that starting a same-document transition before revealing the document would cancel a pending cross-document transition.
+
+		1. Let |transition| be a new {{ViewTransition}} in |document|'s [=relevant Realm=],
```

Comment:
nit: Should we move this after resolving the rule below? Other |transition| is unused.